### DISCUSSION TITLE
Slimserver plugin would be great, Qobuz will end up taking clients

### DISCUSSION THREAD

**Discussion by baslking** [tidal-music/discussions#42] [Ideas & Feature requests]
Logitech is shutting down their streaming service for Squeezeplayer streamers.  Tidal works on these players but requests are proxied rather than using the API directly. Qobuz has a plugin that works directly on Slimserver.  Logitech is suggesting that the solution is to quit Tidal and move to Qubuz, which is unfortunate.  The plugin is likely similar

**Comment by Hermano2** [comment]
Wish someone would take this and run with it. Many Logitech media server users who use Tidal are now having to leave...most heading to Qobuz/Deezer/Spotify as MySqueezebox.com servers being shut down.

Should be simple if Tidal allow....to build LMS plugin for Tidal....similar to Qobuz etc.

**Comment by real-t0mg** [comment]
I'm one of the users affected by Logitech's shutdown of their MySqueezebox.com service. I run my own Logitech Media server locally on Slackware Linux since 2009. I'm not a coder but can help test/QA a plugin that get's Tidal running locally on the squeezebox server.

**Comment by stefanmoro** [comment]
Hey,

Thank you all for the input. I am personally not that familiar with Logitech Media Server/Slimserver and exactly what would be required to build a TIDAL plugin/integration for it.

Since TIDAL Developer Platform does not currently support all the functionality needed, I suggest reaching out to [TIDAL support](https://support.tidal.com), asking about this particular integration and if there are any plans to update it.
